lib/galaxy/managers/users.py

```diff
@@ -499,8 +498,11 @@ def send_activation_email(self, trans, email, username):
         Send the verification email containing the activation link to the user's email.
         """
         activation_token = self.__get_activation_token(trans, email)
-        activation_link = url_for(
-            controller="user", action="activate", activation_token=activation_token, email=escape(email), qualified=True
+        activation_link = trans.url_builder(
+            "/activate",
```

Comment:
Should this be:
```suggestion
            "/user/activate",
```
?

Reply:
Surprisingly both links work:
- `http://127.0.0.1:8081/activate?activation_token={token}&email={email}`
- `http://127.0.0.1:8081/user/activate?activation_token={token}&email={email}`

The previous `url_for` was generating the one without `/user`, but if both work I think is more meaningful to generate the one with `/user` as you suggested :+1: